src/models/dish.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+// Define the Dish schema
+const dishSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  cookerId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Cooker',
+  },
+  description: String,
+  image: String,
+  ingredients: [String],
+  price: {
+    type: Number,
+    required: true,
+  },
+  reviews: [
```

Comment:
The 'review' model should be imported and used like this:
const Review = require('review')
reviews: [review]